theme/themepacks.less

```diff
@@ -0,0 +1,62 @@
+/**************************************************************************
+    CSS theme variables
+    These are overridden by targets and used in: react-common, CRA webapps.
+    To add new theme variable:
+        1. Add variable to :root and .high-contrast below
+        2. Set the variable value in each target's theme/themepacks.less
+***************************************************************************/
+
+// TODO: Define the actual set of theme variables we want to support

```

Comment:
The CSS variables defined here are placeholder. I have a TODO to comb the editor css and extract the needed variables to here.

Reply:
While combing, I will also be referencing vscode's theme system to see what we can adopt from their schema. We probably have overlap, might be good to align where we can.